pkg/kubeapiserver/options/authentication.go

```diff
@@ -134,7 +134,8 @@ type ServiceAccountAuthenticationOptions struct {
 	ExtendExpiration bool
 	// OptionalTokenGetter is a function that returns a service account token getter.
 	// If not set, the default token getter will be used.
-	OptionalTokenGetter func(factory informers.SharedInformerFactory) serviceaccount.ServiceAccountTokenGetter
+	OptionalTokenGetter      func(factory informers.SharedInformerFactory) serviceaccount.ServiceAccountTokenGetter
+	ExternalPublicKeysGetter serviceaccount.PublicKeysGetter
```

Comment:
add godoc for this field, is this mutually exclusive with KeyFiles?

Reply:
Done